test/dependency-manager/volumes.test.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from '@oclif/test';
+import axios from 'axios';
+import mock_fs from 'mock-fs';
+import sinon from 'sinon';
+import Build from '../../src/commands/build';
+import LocalDependencyManager from '../../src/common/dependency-manager/local-manager';
+import * as DockerCompose from '../../src/common/docker-compose';
+import DockerComposeTemplate from '../../src/common/docker-compose/template';
+
+describe('volumes', function () {
+  let compose: DockerComposeTemplate;
+
+  beforeEach(async () => {
+    // Stub the logger
+    sinon.replace(Build.prototype, 'log', sinon.stub());
+
+    const backend_json = {
+      "name": "architect/backend",
+      "parameters": {
+        "VOLUME_PATH": {
+          "default": '/tmp/volume_path'
+        }
+      },
+      "volumes": {
+        "service_config_volume": {
+          "mountPath": "/tmp/volume"
+        },
+        "volume_from_param": {
+          "mountPath": "$VOLUME_PATH"
+        },
+        "split_volume": {
+          "mountPath": "/tmp:/vol/service_config"
```

Comment:
What's the case for putting a split into a mount path? Splits are the joining of a host path to a container path, and the service config won't have any knowledge of the host. Splits should be created as a result of joining a service and environment config.

Reply:
Some of the code that Jon added looked like it expected something like this to be a possibility. I think I'm misunderstanding how you want the env config used overall though and how that will map to compose values, so let's have a short conversation about that tomorrow

Reply:
The volumes will be composed by combining the host location from the environment config with the container location from the service config. Sounds like you've figured it out though.